src/pages/SignUp.jsx

```diff
@@ -122,6 +128,7 @@ const SignUp = () => {
                 name="password"
                 value={password}
                 onChange={handleOnChange}
+                placeholder="password has to be more than 6 characters"
```

Comment:
It would be better if you put this below the input box. Here the placeholder should be something like Enter password

Reply:
That was you can keep the page consistent. (As the placeholder in the other input box is **confirm password**